newscoop/src/Newscoop/NewscoopBundle/Security/Http/Authentication/FrontendLogoutSuccessHandler.php

```diff
@@ -50,4 +51,15 @@ public function onLogoutSuccess(Request $request)
 
         return parent::onLogoutSuccess($request);
     }
+
+    private function setTargetUrl(Request $request)
```

Comment:
This `one liner` is not enough.

what if you won't provide the `url` and `_target_path` - it will assign `null` to `$this->targetUrl` variable, which will result in an exception, because you can't redirect to null route (null should be replaced with `$this->targetUrl` at least):

```php
$this->targetUrl = $request->query->get('_target_path', $request->query->get('url', $this->targetUrl));
```

Then in this case, why to bother and assign a value to `$this->targetUrl` at all, if there is no `url` or `_target_path` provided at all? (note that `targetUrl` var will always have a default value as its required) - just to assign the same value to `$this->targetUrl` ?

Isn't it for what the private functions are meant for, also to separate the parts of code ?